packages/pigeon/example/app/ios/Runner/AppDelegate.swift

```diff
@@ -37,16 +37,67 @@ private class PigeonFlutterApi {
   }
 
   func callFlutterMethod(
-    aString aStringArg: String?, completion: @escaping (Result<String, Error>) -> Void
+    aString aStringArg: String?, completion: @escaping (Result<String, PigeonError>) -> Void
   ) {
     flutterAPI.flutterMethod(aString: aStringArg) {
-      completion(.success($0))
+      completion($0)
     }
   }
 }
 // #enddocregion swift-class-flutter
 
-@UIApplicationMain
+// #docregion swift-class-event
+class EventListener: StreamEventsStreamHandler {
+  var eventSink: PigeonEventSink<SealedBaseClass>?
+
+  override func onListen(withArguments arguments: Any?, sink: PigeonEventSink<SealedBaseClass>) {
+    eventSink = sink
+  }
+
+  func onIntEvent(event: Int64) {
+    if let eventSink = eventSink {
+      eventSink.success(IntEvent(data: event))
+    }
+  }
+
+  func onStringEvent(event: String) {
+    if let eventSink = eventSink {
+      eventSink.success(StringEvent(data: event))
+    }
+  }
+
+  func onEventsDone() {
+    eventSink?.endOfStream()
+    eventSink = nil
+  }
+}
+// #enddocregion swift-class-event
+
+func sendEvents(_ eventListener: EventListener) {
+  var timerActive = false
+  var timer: Timer?
+  var count: Int64 = 0
+  if !timerActive {
+    timerActive = true
```

Comment:
This method activates 2 times on startup, this is to prevent that. I'm not certain why, maybe there's a better way, maybe it doesn't matter, maybe it just shouldn't be in the example.

Reply:
How does a local variable change the behavior of multiple calls to the method?

Reply:
It was a problem before I refactored the code here, let me check again.